hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestRegionReplicaReplicationEndpoint.java

```diff
@@ -487,4 +490,26 @@ private void testRegionReplicaReplicationIgnores(boolean dropTable, boolean disa
       connection.close();
     }
   }
+
+  private void createOrEnableTableWithRetries(TableDescriptor htd, boolean createTableOperation) {
+    // Helper function to run create/enable table operations with a retry feature
+    boolean continueToRetry = true;
+    int tries = 0;
+    while (continueToRetry && tries < 100) {
```

Comment:
Although we don't expect `ReplicationException` to keep showing up till 100s, but 100s is huge duration that we are waiting on here. It will make tests timeout if we have some issue. Let's keep this 50 maybe?

Reply:
Good point, I will change it 👍